walkthroughs/week-0-setup/intro-walkthrough.md

```diff
@@ -7,6 +7,8 @@ You can return to this walkthrough anytime by executing this command:
 
 ```bash
 teachme ~/software-product-sprint/walkthroughs/week-0-setup/intro-walkthrough.md
+``````bash
+teachme ~/software-product-sprint/walkthroughs/week-0-setup/intro-walkthrough.md
```

Comment:
What does this do?

Reply:
I think I accidentally committed all the files instead of the ones that were modified

Reply:
Can you fix this extra line?

Reply:
I think I fixed it